cdap-docs/admin-manual/source/index.rst

```diff
@@ -69,6 +69,13 @@ transaction service maintenance, troubleshooting, and introduces the CDAP UI.**
 
 - |metrics|_ CDAP collects **metrics about the application’s behavior and performance**.
   
+.. |monitoring| replace:: **Monitoring:**
+.. _monitoring: installation/monitoring.html
+
+- |monitoring|_ CDAP collects **logs and metrics** for all of its internal services. 
+  This section provides links to the relevant APIs for accessing these logs and metrics,  
+  and to support for external monitoring tools.
```

Comment:
"provides links to support for..." sounds a little wierd to me.  How about something like: "This section provides links to the relevant APIs for accessing these logs and metrics, and an overview of available monitoring utilities."